.github/workflows/check-manifest.yml

```diff
@@ -0,0 +1,23 @@
+name: "Check Manifest"
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        # scipy and cython are required to build sdist
+        run: |
+          python -m pip install --upgrade pip
+          pip install check-manifest scipy cython
+        shell: bash
```

Comment:
I do not think we need to specific the shell here, because it defaults to bash on ubuntu

```suggestion
```